src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/prison/UserController.kt

```diff
@@ -31,7 +33,62 @@ import javax.validation.constraints.NotEmpty
 @Validated
 class UserController(
   private val nomisUserService: UserService,
+  @Value("\${application.smoketest.enabled}") private val smokeTestEnabled: Boolean,
```

Comment:
Still not sure we need this. Does the smoketest ever call manage users?

Reply:
I don't actually know - this property is used in a similar way on the other amend user email end point, so assumed this would also be needed. Where would I need to look to confirm? Maybe best to cover both of those off with a separate ticket to check it and either leave or remove them?

Reply:
Raising separate ticket to cover this